chainforge/react-server/src/ModelSettingSchemas.js

```diff
@@ -1000,6 +1000,79 @@ const AlephAlphaLuminousSettings = {
   },
 };
 
+const OllamaSettings = {
+  fullName: "Ollama",
+  schema: {
+    type: "object",
+    required: ["shortname"],
+    properties: {
+      shortname: {
+        type: "string",
+        title: "Nickname",
+        description:
+          "Unique identifier to appear in ChainForge. Keep it short.",
+        default: "Ollama",
+      },
+      ollamaModel: {
```

Comment:
I would have named this field `model` but it looks like `model` is a special attribute that's used to determine the provider. Since I didn't want to add all the models supported by Ollama to the `NativeLLM` enum, I decided to go with a different name and having `call_ollama_provider` map `ollamaModel` to the `model` key in the API request body.